sonarqube/README.md

```diff
@@ -0,0 +1,55 @@
+# Agent Check: SonarQube
+
+## Overview
+
+This check monitors [SonarQube][1].
+
+## Setup
+
+### Installation
+
+The SonarQube check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `sonarqube.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your sonarqube performance data.
+   See the [sample sonarqube.d/conf.yaml][2] for all available configuration options.
+
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page.
+   You can specify the metrics you are interested in by editing the configuration below.
+   To learn how to customize the metrics to collect visit the [JMX Checks documentation][3] for more detailed instructions.
+   If you need to monitor more metrics, contact [Datadog support][4].
```

Comment:
Maybe add a section explaining to configure two instances to collect JMX and API metrics